setup.cfg

```diff
@@ -1,3 +1,47 @@
+[metadata]
+name = ctdcal
+author =
+    Joseph Gum
+    Andrew Barna
+    Michael Kovatch
+author-email = jgum@ucsd.edu
+home-page = https://github.com/cchdo/ctdcal
+description = CTD and bottle data processing package from UCSD ODF
+long_description = file: README.md
+license_file = file: LICENSE
+platform = any
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Topic :: Scientific/Engineering
+keywords =
+    oceanography
+    seawater
+    TEOS-10
+    ctd
+    calibration
+
+[options]
+python_requires = >=3.5
+packages = ctdcal
+install_requires =
+    numpy
+    matplotlib
+    scipy
+    pandas
```

Comment:
Because pandas is such a jerk about backwards compatibility, it might be worthwhile to define an exact version that works right now, but it isn't something I feel strongly about.

Reply:
Do you know how this compares to definitions in `requirements.txt`?

Reply:
It's almost the same in how it works... see these "pinned" deps in one of the other projects:
https://github.com/cchdo/hydro/blob/master/setup.cfg#L42-L51

Reply:
So then that replaces the need for the requirements file?

Reply:
That's the way I use it yes it replaces the requirements file... a `pip install -e '.[complete]'` and everything gets installed.